tests/gui/modules/drive-scanner.spec.js

```diff
@@ -332,7 +367,8 @@ describe('Browser: driveScanner', function () {
                 path: 'H:'
               }
             ],
-            system: false
+            isSystem: false,
+            isRemovable: true
```

Comment:
I'm wondering why you've added `isRemovable` to all these test-cases, given that nothing else even looks at this property?